Fix build with libheif >=1.18.0
See: https://github.com/strukturag/libheif/commit/cf0d89c6e0809427427583290547a7757428cf5a

Add new librsvg GdkPixbuf loader name.

Index: configure
--- configure.orig
+++ configure
@@ -30412,18 +30412,14 @@ $as_echo "yes" >&6; }
 fi
 fi
 
-can_import_heic=no
-can_export_heic=no
-can_import_avif=no
-can_export_avif=no
+can_import_heic=yes
+can_export_heic=yes
+can_import_avif=yes
+can_export_avif=yes
 if test "x$have_libheif" = xyes; then
-  can_import_heic=`$PKG_CONFIG --variable=builtin_h265_decoder libheif`
-  can_export_heic=`$PKG_CONFIG --variable=builtin_h265_encoder libheif`
   if test "x$can_import_heic" = xyes; then
     MIME_TYPES="$MIME_TYPES;image/heif;image/heic"
   fi
-  can_import_avif=`$PKG_CONFIG --variable=builtin_avif_decoder libheif`
-  can_export_avif=`$PKG_CONFIG --variable=builtin_avif_encoder libheif`
   if test "x$can_import_avif" = xyes; then
     MIME_TYPES="$MIME_TYPES;image/avif"
   fi
@@ -33906,7 +33902,8 @@ fi
     # So let's test files the shell way.
     if (test "x$platform_win32" = "xyes" &&
         test -f "$gdk_pixbuf_moduledir/libpixbufloader-svg.dll") ||
-       test -f "$gdk_pixbuf_moduledir/libpixbufloader-svg.so"; then
+       test -f "$gdk_pixbuf_moduledir/libpixbufloader-svg.so" ||
+       test -f "$gdk_pixbuf_moduledir/libpixbufloader_svg.so"; then
       # We must not use $PKG_CONFIG nor PKG_CHECK_* macros because we need
       # to make sure we use the native pkg-config (in case we cross-compile).
       if pkg-config --atleast-version=2.56.2 glib-2.0 &&
